### PR TITLE
[#234] Use real models in EDI tests

### PR DESCRIPTION
**Pull request by ablatov**
- tag by model_version EDI tests
- use real Test-Summation and Test-Summation-v1.1 models from examples for tests
- remove test-bare models
- grab a model image from Jenkins job log
- set timeouts for tests, and stuck keywords (undeploy with version)

### CODE REVIEW

**Comment by ablatov** [comment]
Test this please

File: examples/Test-Summation-v1.1/performance/test_basic.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#
+#    Copyright 2017 EPAM Systems
```
Comment (aliaksandr-d) [inline examples/Test-Summation-v1.1/performance/test_basic.py]: Current year is 2018

File: legion/legion/utils.py [hunk 2]
```diff
@@ -450,8 +450,8 @@ def send_header_to_stderr(header, value):
     :type value: str
     :return: None
     """
-    message = 'X-Legion-%s:%s' % (header, value)
-    print(message, file=sys.__stderr__, flush=True)
+    message = 'X-Legion-{}:{}'.format(header, value)
```
Comment (aliaksandr-d) [inline legion/legion/utils.py]: Why dont you add *\n* here instead of next line?

File: tests/robot/4_check_edi.robot [hunk 3]
```diff
@@ -58,11 +62,11 @@ Check EDI deploy with scale to 1
 Check EDI deploy with scale to 2
```
Comment (aliaksandr-d) [inline tests/robot/4_check_edi.robot]: Can we check actual scale here?

File: tests/robot/5_check_edi_milti_versioned_models.robot [hunk 4]
```diff
@@ -1,11 +1,17 @@
 *** Settings ***
```
Comment (aliaksandr-d) [inline tests/robot/5_check_edi_milti_versioned_models.robot]: Please, fix the name of this file

File: tests/robot/4_check_edi.robot (hunk 3 above)
Comment (ablatov) [reply tests/robot/4_check_edi.robot]: @aliaksandr-d what do you mean? We deploy with scale = 2 flag, and check by "inspect" command  that it scaled to 2 after deploy.

**Pull request closed without merging**